src/diffusers/pipelines/latent_diffusion/pipeline_latent_diffusion_superresolution.py

```diff
@@ -66,7 +66,7 @@ def __init__(
     @torch.no_grad()
     def __call__(
         self,
-        init_image: Union[torch.Tensor, PIL.Image.Image],
+        image: Union[torch.Tensor, PIL.Image.Image],
```

Comment:
we will have to deprecate `init_image` here to not surprise the user here too much. Could you implement the same logic we're using for schedulers for all pipelines: 
https://github.com/huggingface/diffusers/blob/5755d16868ec3da7d5eb4f42db77b01fac842ea8/src/diffusers/pipelines/ddpm/pipeline_ddpm.py#L76

Here would should do the following in addition to renaming `init_image` to `image`:
1. Add a `**kwargs` argument to the function call
2. 
```python
 message = (
    "Please make sure to use `image` instead of `init_image`."
)
init_image = deprecate("init_image", "0.12.0", message, take_from=kwargs)
image = init_image or image
```

Reply:
We should do this for all pipelines :-)